ui/de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -654,6 +655,11 @@ void restoreWindows(List<WindowState> windowStates) {
         for (WindowState ws : windowStates) {
             desktopWindowManager.show(ws);
         }
+        if (userInfo.hasAgaveRedirect()) {
+            AgaveAuthPrompt prompt = AgaveAuthPrompt.getInstance();
+            prompt.show();
+            desktopWindowManager.stickWindowToTop(prompt);
```

Comment:
I added this specifically for the case when a user is saving their sessions.  When they first log in, all the windows they had open will reopen and can hide the prompt.  This makes sure the prompt is in focus for the user to interact with first.